extensions/wikia/GlobalNavigation/styles/GlobalNavigationAccountNavigation.scss

```diff
@@ -25,12 +25,15 @@
 		border: 0;
 	}
 
-	.account-navigation-first-item {
+	.account-navigation-item {
 		list-style: none;
 
 		.links-container {
 			display: table-row;
 			height: $global-navigation-height-without-margin;
+			font-size: 12px;
```

Comment:
The same font-size many times in the same file, what do you think about using variable?